### PR TITLE
added a few books

### PR DESCRIPTION
**Pull request by kri88**
## Hacktoberfest notes:

- due to volume of submissions, we may not be able to review PRs that do not pass tests and do not have informative titles.
- please read our [contributing guidelines](/CONTRIBUTING.md)
- be sure to check the output of Travis-CI for linter errors
- if this is your first open source contribution, make sure it's not your last!

## What does this PR do?
Add Resource(s) | Remove Resource(s) | Add info | Improve Repo

### Checklist:
- [x] Not a duplicate
- [x] Included author(s) if appropriate
- [x] Lists are in alphabetical order
- [x] Needed indications added (PDF, access notes, under construction)

### CODE REVIEW

File: free-programming-books-pl.md [hunk 1]
```diff
@@ -81,6 +81,7 @@
 ### CSS
 
 * [Kaskadowe Arkusze Stylów](http://www.kurshtml.edu.pl/css/index.html) - Sławomir Kokłowski
+* [Kurs CSS](http://www.kurshtmlcss.pl/kurs-css/)
```
Comment (eshellman) [inline free-programming-books-pl.md:84]: consider putting this in courses

File: free-programming-books-pl.md [hunk 2]
```diff
@@ -93,6 +94,7 @@
 
 * [HTML dla zielonych](http://www.kurshtml.edu.pl/html/zielony.html) - Sławomir Kokłowski
 * [KURS HTML](http://www.kurshtml.edu.pl) - Sławomir Kokłowski
+* [KURS HTML/CSS](http://www.kurshtmlcss.pl/kurs-html/)
```
Comment (eshellman) [inline free-programming-books-pl.md:97]: same here

File: free-programming-books-pl.md [hunk 3]
```diff
@@ -123,6 +125,7 @@
 ### MySQL
 
 * [Bazy Danych MYSQL](https://miroslawzelent.pl/kurs-mysql/)
+* [Kurs MYSQL](http://webmade.org/kursy-online/kurs-mysql.php)
```
Comment (eshellman) [inline free-programming-books-pl.md:128]: hint - if the title is "kurs-msql" maybe it's a "kurs"

File: free-programming-books-pl.md [hunk 4]
```diff
@@ -167,5 +170,6 @@
 
 ### SQL
 
+* [Kurs SQL](https://www.sqlpedia.pl/kurs-sql/) - Jakub Kasprzak
```
Comment (eshellman) [inline free-programming-books-pl.md:173]: same

**Review by eshellman — CHANGES_REQUESTED** [review]

File: free-programming-books-pl.md (hunk 1 above)
Comment (kri88) [reply free-programming-books-pl.md:84]: ok

File: free-programming-books-pl.md (hunk 3 above)
Comment (kri88) [reply free-programming-books-pl.md:128]: ok

File: free-programming-books-pl.md (hunk 4 above)
Comment (kri88) [reply free-programming-books-pl.md:173]: ok

**Comment by kri88** [comment]
closed

**Pull request closed without merging**